### PR TITLE
added logout feature

### PR DESCRIPTION
**Pull request by Tauqeer-Ahmed-99**
added logout feature

### CODE REVIEW

File: src/redux/actions/auth.action.js [hunk 1]
```diff
@@ -39,3 +39,12 @@ export const login = () => async (dispatch) => {
     });
   }
 };
+
+export const logout = () => async (dispatch) => {
+  await auth.signOut;
```
Comment (faran4engg) [inline src/redux/actions/auth.action.js]: Is this not a function?

File: src/components/sideBar/SideBar.jsx [hunk 2]
```diff
@@ -40,7 +48,7 @@ const SideBar = ({ onToggleSideBar, handleToggleSideBar }) => {
         <span>Dislikes</span>
       </li>
       <hr />
-      <li>
+      <li onClick={logoutHandler}>
```
Comment (faran4engg) [inline src/components/sideBar/SideBar.jsx:51]: I would do handleLogout

Comment (Tauqeer-Ahmed-99) [reply src/components/sideBar/SideBar.jsx:51]: I am using this convention from starting

File: src/redux/actions/auth.action.js (hunk 1 above)
Comment (Tauqeer-Ahmed-99) [reply src/redux/actions/auth.action.js]: yes I saw it now, but I checked my app and its working fine without these paranthesis @faran4engg

**Pull request merged**